lib/CPathfinder.cpp

```diff
@@ -318,23 +341,30 @@ void CPathfinder::initializeGraph()
 			for(pos.z=0; pos.z < out.sizes.z; ++pos.z)
 			{
 				const TerrainTile *tinfo = &gs->map->getTile(pos);
-				CGPathNode &node = graph[pos.x][pos.y][pos.z];
-				node.accessible = evaluateAccessibility(pos, tinfo);
-				node.turns = 0xff;
-				node.moveRemains = 0;
-				node.coord = pos;
-				node.land = tinfo->terType != ETerrainType::WATER;
-				node.theNodeBefore = nullptr;
+				switch (tinfo->terType)
+				{
+					case ETerrainType::WRONG:
+					case ETerrainType::BORDER:
+					case ETerrainType::ROCK:
```

Comment:
Those aren't unitialized (object created by CPathsInfo), but just not checked for accessibility as this is my way of avoiding usage of those layers. What I gonna do is rename "initializeNode" to something more else...